db/migrate/20180601160100_add_deb_version_cmp_function_to_db.rb

```diff
@@ -0,0 +1,148 @@
+class AddDebVersionCmpFunctionToDb < ActiveRecord::Migration[4.2]
```

Comment:
Can this db function and any related code be moved to https://github.com/Katello/katello/pull/8277? That will help move this PR forward and #8277 can truly be focused on applicability which is bleeding into this PR

Reply:
I will see if I can untie some knots between these PRs :smile: